starport/pkg/cosmosanalysis/module/module_test.go

```diff
@@ -2,11 +2,27 @@ package module
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/kr/pretty"
 )
 
 func TestExampleDiscover(t *testing.T) {
-	pretty.Println(Discover(context.Background(), "/home/ilker/Documents/code/src/github.com/tendermint/starport/local_test/mars"))
+	goPath := os.Getenv("GOPATH")
+	t.Run("test folder without proto files", func(t *testing.T) {
+		pretty.Println(Discover(
+			context.Background(),
+			filepath.Join(goPath, "src/github.com/tendermint/starport/local_test/mars"),
+			"proto",
+		))
+	})
+	t.Run("test folder with proto files", func(t *testing.T) {
```

Comment:
```suggestion
```

`Discover` is for discovering modules and google/protobuf is not an sdk module, I think we can delete this one.